aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move

```diff
@@ -20,6 +20,7 @@ module hello_blockchain::message {
     /// There is no message present
     const ENO_MESSAGE: u64 = 0;
 
+    #[view]
```

Comment:
I think we have a tutorial uses this file, do we need to update something there also?